packages/semaphore-signature-pcd/test/SemaphoreSignature.spec.ts

```diff
@@ -57,7 +57,7 @@ describe("semaphore signature should work", function () {
     const { prove, verify } = SemaphoreSignaturePCDPackage;
     // change merkle root to make it invalid
     const pcd = await prove(args);
-    pcd.proof.proof.merkleTreeRoot = "0";
+    pcd.proof[0] = "1";
     const verified = await verify(pcd);
     assert.equal(verified, false);
   });
```

Comment:
A lot of the following tests are a bit unnecessary now that all of the data that used to be in the proof has been moved into the claim. We can just have tests that test changing different parts of the claim and making sure the proof is not verified, like what I suggested for the SemaphoreGroupPCD.

Reply:
OK, i've created a test for each of the fields in `claim`, and made sure that tampering with those fields results in a PCD that does not verify.